cli/azd/internal/repository/app_init.go

```diff
@@ -1107,12 +1112,61 @@ func promptSpringBootVersion(console input.Console, ctx context.Context) (string
 	}
 }
 
+func promptMissingEventHubsNameOrExit(console input.Console, ctx context.Context, eventHubs *appdetect.AzureDepEventHubs) {
+	switch eventHubs.FromDependency {
+	case appdetect.SpringIntegrationEventHubs, appdetect.SpringMessagingEventHubs, appdetect.SpringKafka:
+		eventHubsName, err := promptEventHubsName(console, ctx)
```

Comment:
Is it possible that we require multiple event hub names here, and it just prompt for one name?

Reply:
It's possible even if we don't show this kind of scenario in our samples. I have updated code to support user to input multiple event hubs names